editable-comments-web/src/safe_api.js

```diff
@@ -184,7 +166,6 @@ export default class SafeApi {
         // The network operation is performed only when we call getEntries for validating that the MutableData exists
         const entriesHandle = await window.safeMutableData.getEntries(mdHandle);
         window.safeMutableDataEntries.free(entriesHandle);
-        window.safeMutableData.free(mdHandle);
```

Comment:
`free()` function is removed from `safeMutableData`, Though it returns 64 bit handle, I don't see any function that free the `mdHandle`.  Is there any way to free this `Mutable Data Handle`?

Reply:
You are right, it was removed, which is not correct as we are still returning a handle when you create a MD, so we should either put the `free` function back or return the MD object instead of a handle. @hunterlester we have an issue here in the DOM API, we can approve this PR so we can publish the updated comments plugin example on Alpha-2 but we'll have to the fix the safe-app plugin and the comments plugin later on to rename the var names as it won't be a handle anymore?